Makefile

```diff
@@ -0,0 +1,42 @@
+.PHONY: test_contract test_contract.rs test-project test all check-ink-wrapper \
+	check-test-project all-dockerized tooling build-builder build-node run-node
+
+build-builder:
+	docker build --tag ink-builder --file ci/Dockerfile.builder ci
+
+build-node:
+	docker build --tag aleph-onenode-chain --file ci/Dockerfile.aleph-node ci
+
+run-node: build-node
+	docker run --detach --rm --network host aleph-onenode-chain
+
+test_contract:
+	cd test_contract && cargo contract build --release
+	cd test_contract && cargo contract upload --suri //Alice --url ws://localhost:9944 || true
+
+test_contract.rs: test_contract
+	cd ink-wrapper && cargo run -- -m ../test_contract/target/ink/test_contract.json \
+		| rustfmt --edition 2021 > ../test-project/src/test_contract.rs
+
+test: test_contract.rs
+	cd test-project && cargo run
+
+check-ink-wrapper:
+	cd ink-wrapper && cargo fmt --all --check
+	cd ink-wrapper && cargo clippy --all-features -- --no-deps -D warnings
+
+check-test-project:
+	cd test-project && cargo fmt --all --check
+	cd test-project && cargo clippy --all-features -- --no-deps -D warnings
+
+all-dockerized: run-node build-builder
+	docker run --rm --network host \
+		--mount type=bind,source="$(shell pwd)",target=/code \
+		--workdir /code \
+		ink-builder \
+		make all
+
+tooling:
+	rustup component add rustfmt clippy
+
+all: tooling check-ink-wrapper check-test-project test
```

Comment:
Can you add a command to kill everything?

Reply:
:+1: